Gemfile

```diff
@@ -20,7 +20,10 @@ gem 'scoped_search', '>= 4.1.8', '< 5'
 gem 'ldap_fluff', '>= 0.5.0', '< 1.0'
 gem 'apipie-rails', '>= 0.5.17', '< 0.6.0'
 gem 'apipie-dsl', '>= 2.2.6'
-gem 'rdoc'
+# Pin rdoc to prevent psych > 4 (https://github.com/ruby/rdoc/commit/ebe185c8775b2afe844eb3da6fa78adaa79e29a4)
+gem 'rdoc', '~> 6.3.3'
```

Comment:
Is this redundant if we already pin `psych < 4`? Or can't bundler resolve that properly?

Reply:
It is redundant, I'm fixing it both ways. bundler would resolve that. Though IMHO it is usefull information to have and not need to dig it again :thinking: